### DISCUSSION TITLE
v2023.10.7

### DISCUSSION THREAD

**Discussion by derme302** [derme302/triple-hyde#11] [Announcements]
## What's Changed
* Correctly breakout about sidebar partial by @derme302 in https://github.com/derme302/triple-hyde/pull/7
* Add threads icon and update Font Awesome by @derme302 in https://github.com/derme302/triple-hyde/pull/8
* Add giscus support by @MarcoIeni in https://github.com/derme302/triple-hyde/pull/10

**Full Changelog**: https://github.com/derme302/triple-hyde/compare/v2023.10.3...v2023.10.7

<hr /><em>This discussion was created from the release <a href='https://github.com/derme302/triple-hyde/releases/tag/v2023.10.7'>v2023.10.7</a>.</em>

**Comment by marcoieni** [comment]
hey great to know there's a fork of this theme. Starred! :)

**Reply by derme302** [reply]
Thanks! I've been maintaining it within my blog for a couple years and decided I should break it out to share
